### PR TITLE
Extend AppShell

### PR DESCRIPTION
**Pull request by wrdx**
Extend AppShell for more control from within app

### CODE REVIEW

File: Console/Command/AclExtrasShell.php [hunk 1]
```diff
@@ -21,7 +21,7 @@
  * @package		acl_extras
  * @subpackage	acl_extras.Console.Command
  */
-class AclExtrasShell extends Shell {
+class AclExtrasShell extends AppShell {
```
Comment (markstory) [inline Console/Command/AclExtrasShell.php]: Not all applications have an `AppShell` though.

Comment (WrDX) [reply Console/Command/AclExtrasShell.php]: Ok, but every Shell in the CakePHP 2.x lib itself extends the AppShell, wouldn't that break stuff too then?

Comment (WrDX) [reply Console/Command/AclExtrasShell.php]: To test this further I've removed the AppShell.php from my app/Console/Command directory and the AclExtras console commands still work fine. I expect this has to do with the AppShell in the lib being used.

Comment (markstory) [reply Console/Command/AclExtrasShell.php]: Oops I forgot this was for 2.x

**Pull request merged**